### PR TITLE
Stats and subactions are responsive and separate components

### PR DESCRIPTION
**Pull request by LiFaytheGoblin**
PR depends on #2108

### Resolves:
Part of #2086: The stats and subactions now position nicely depending on media queries. They are also put in separate components.

### Test Coverage:
MacOS, Chrome
npm test passes

### CODE REVIEW

**Comment by LiFaytheGoblin** [comment]
Argh oh no! I accidentally also took in some commits for the report modal ._. Can these just be left out when merging or so? @rschamp

File: src/views/preview/preview.scss [hunk 1]
```diff
@@ -539,8 +359,3 @@ $medium-and-small: "screen and (max-width : #{$tablet}-1)";
         }
     }
 }
-
-.report-text textarea {
```
Comment (LiFaytheGoblin) [inline src/views/preview/preview.scss]: This will look fine if #2108 goes in first @rschamp

Comment (rschamp) [reply src/views/preview/preview.scss]: Sure — please update the description to say that this PR depends on #2108.

Comment (LiFaytheGoblin) [reply src/views/preview/preview.scss]: done!

File: src/views/preview/subactions.scss [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+            //         background-color: $active-gray;
+            //     }
+            //
+            //     &:active {
+            //         border: 0 solid transparent;
+            //         box-shadow: inset 0 0 5px $box-shadow-gray;
+            //         background-color: $active-dark-gray;
+            //         padding: calc(.75em + 1px) calc(1.5em + 1px);
+            //     }
+            // }
```
[the remark this answers is not in the document]
Comment (LiFaytheGoblin) [reply src/views/preview/subactions.scss]: yep it was like this already

**Comment by LiFaytheGoblin** [comment]
Okay I am kind of worried about merging now @Ray: It said I had merging problems, then locally I merged develop into my branch and resolved the issues which was tricky but it worked. Then I pushed and then it said again that I had merging issues?! just clicked on fix issues or so and copy pasted what I had locally. Is that an okay thing to do?

**Pull request merged**